common/src/main/java/org/dash/wallet/common/ui/enter_amount/EnterAmountFragment.kt

```diff
@@ -77,6 +86,13 @@ class EnterAmountFragment: Fragment(R.layout.fragment_enter_amount) {
         val args = requireArguments()
         binding.maxButtonWrapper.isVisible = args.getBoolean(ARG_MAX_BUTTON_VISIBLE)
         binding.amountView.showCurrencySelector = args.getBoolean(ARG_SHOW_CURRENCY_SELECTOR_BUTTON)
+        binding.amountView.hideResultContainer = args.getBoolean(ARG_HIDE_AMOUNT_RESULT_CONTAINER)
+        binding.currencyOptions.isVisible = args.getBoolean(ARG_CURRENCY_Code)
```

Comment:
This seems to be an error. It should be `ARG_CURRENCY_OPTIONS_PICKER_VISIBLE` here. This causes the currency option picker to be hidden in other Enter Amount screens.

Reply:
Fixed